<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PABA Training Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;                      /* Text color */
    border-radius: 8px;                /* Rounded corners */
    padding: 6px 12px;                 /* Padding inside button */
font: 12pt &quot;Bahnschrift SemiCondensed&quot;;
    font-weight: bold;                 /* Bold text */
    border: 2px solid #fff;         /* Border with a slightly darker shade */
}


QPushButton:pressed {
       /* Darken the border further when pressed */
    padding-left: 14px;                /* Slight movement for press effect */
    padding-top: 6px;                  /* Slight movement for press effect */
}

QLabel {
font: 12pt &quot;Bahnschrift SemiCondensed&quot;;
    font-weight: bold;   
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="MainScreen">
      <property name="styleSheet">
       <string notr="true">#loginFrame{
background-color:#EFEFEF;
}
QLabel
{
color:white;
font-style:Century Gothic;
font-size:12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="sideBar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #48ACAC;
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>180</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="resources.qrc">:/images/images/logo.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="fameSideBarButtons">
            <property name="styleSheet">
             <string notr="true">QPushButton:hover {
    background-color: #2F958D;         /* Slightly darker color on hover */
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>30</number>
             </property>
             <property name="leftMargin">
              <number>11</number>
             </property>
             <property name="topMargin">
              <number>40</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnStartTraining">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Start Training</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUserProfile">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>User Profile</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnLogout">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Log Out</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>1</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    color: white;                      /* Text color */
    border-radius: 4px;                /* Rounded corners */
    padding: 5px 8px;                 /* Padding inside button */
font: 10pt &quot;Bahnschrift SemiCondensed&quot;;
    background-color: #48ACAC;         /* Slightly darker color on hover */
       /* Border with a slightly darker shade */
}

QPushButton:hover {
    background-color: #2F958D;         /* Slightly darker color on hover */
}

QPushButton:pressed {
       /* Darken the border further when pressed */
    padding-left: 14px;                /* Slight movement for press effect */
    padding-top: 6px;                  /* Slight movement for press effect */
}
#btnResetSensor
{

background-color: #986262;
color: white;
border-radius: 4px; 
font: 10pt &quot;Bahnschrift SemiCondensed&quot;;

}
#btnResetSensor::hover
{
background-color:#AE4D4D;


}
#btnResetUser
{

background-color: #986262;
color: white;
border-radius: 4px; 
font: 10pt &quot;Bahnschrift SemiCondensed&quot;;

}
#btnResetUser::hover
{
background-color:#AE4D4D;


}




QLabel {
font: 10pt &quot;Bahnschrift&quot;;
color:black
}
QLineEdit {
background-color:white;
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:black;
padding-bottom:1px;
padding-left:3px;
}

</string>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="page_4">
                <layout class="QVBoxLayout" name="verticalLayout_15">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QFrame" name="frame_17">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_16">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>11</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Bahnschrift</family>
                        <pointsize>14</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color:black;
font: 14pt ;
</string>
                      </property>
                      <property name="text">
                       <string>Training Page</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_16">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_22">
                    <item>
                     <widget class="QFrame" name="frame_21">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_21">
                       <property name="spacing">
                        <number>10</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>20</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item alignment="Qt::AlignRight">
                        <widget class="QPushButton" name="btnViewAccountHistory">
                         <property name="minimumSize">
                          <size>
                           <width>160</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="cursor">
                          <cursorShape>PointingHandCursor</cursorShape>
                         </property>
                         <property name="text">
                          <string>View Account History</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QTableWidget" name="trainingDataTable">
                         <property name="editTriggers">
                          <set>QAbstractItemView::NoEditTriggers</set>
                         </property>
                         <property name="alternatingRowColors">
                          <bool>true</bool>
                         </property>
                         <property name="gridStyle">
                          <enum>Qt::SolidLine</enum>
                         </property>
                         <property name="sortingEnabled">
                          <bool>false</bool>
                         </property>
                         <property name="cornerButtonEnabled">
                          <bool>true</bool>
                         </property>
                         <attribute name="horizontalHeaderMinimumSectionSize">
                          <number>60</number>
                         </attribute>
                         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                          <bool>true</bool>
                         </attribute>
                         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                          <bool>true</bool>
                         </attribute>
                         <row>
                          <property name="text">
                           <string>No Data</string>
                          </property>
                         </row>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_5">
                <layout class="QVBoxLayout" name="verticalLayout_17">
                 <item>
                  <widget class="QFrame" name="frame_18">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <property name="spacing">
                     <number>12</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_18">
                      <property name="text">
                       <string>Score: </string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_19">
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_20">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_19">
                    <item>
                     <widget class="QLabel" name="label_20">
                      <property name="text">
                       <string>Images</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_19">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_18">
                    <item>
                     <widget class="QLabel" name="label_21">
                      <property name="text">
                       <string>Videos</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
